experimental/packages/api-logs/src/types/LoggerOptions.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface LoggerOptions {
+  /**
+   * The schemaUrl of the tracer or instrumentation library
+   * @default ''
+   */
+  schemaUrl?: string;
+
+  /**
+   * The domain for the events created
+   * @default ''
+   */
+  eventDomain?: string;
+
+  /**
+   * Specifies whether the Trace Context should automatically be passed on to the events and logs created by the Logger
+   * @default false
+   */
+  includeTraceContext?: boolean;
+
+  /**
+   * The instrumentation scope attributes to associate with emitted telemetry
+   */
+  attributes?: Attributes;
```

Comment:
Maybe better to use a name like `scopeAttributes` which makes it clear what this is

Reply:
This is being specified [here](https://github.com/open-telemetry/opentelemetry-specification/pull/2676/files#diff-e2981506e39a9bc2c9bcfcf188c5a80542e8b6fd115cf9daf98ba8a1cbda80beR94-R95). @scheler Any thoughts whether it would be better to call this `scopeAttributes`?

Reply:
Sure, that makes it more clear.